librb/src/commio.c

```diff
@@ -357,6 +357,9 @@ rb_setsockopt_reuseaddr(rb_fde_t *F)
 }
 
 #ifdef HAVE_LIBSCTP
+#ifndef SOL_SCTP
+#define SOL_SCTP 132
+#endif
 static int
```

Comment:
Please insert blank lines before and after this ifndef block for enhanced readability.

Reply:
Done, thanks!